### PR TITLE
Add myself to release team

### PR DESCRIPTION
**Pull request by terrytangyuan**
Need this permission to manage releases.

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

**Comment by terrytangyuan** [comment]
@jlewi Could you take a look at this to see if I have sufficient permissions to help out with v1.1 release?

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -513,6 +514,7 @@ orgs:
             - kunmingg
             - zhenghuiwang
             - gabrielwen
+            - terrytangyuan
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml:517]: Please alphabetize the team members

Comment (terrytangyuan) [reply github-orgs/kubeflow/org.yaml:516]: Thanks. Alphabetized.

**Comment by terrytangyuan** [comment]
@jlewi It seems like [kubeflow/release-team](https://github.com/orgs/kubeflow/teams/release-team/members) is not synced with the members specified in org.yaml and I haven't been added to that team automatically. Could you look into it?

**Pull request merged**